### DISCUSSION TITLE
Obvious first question: Open-source?

### DISCUSSION THREAD

**Discussion by fritzalder** [lucaapp/security-concept#5] [Q&A] [answered]
Sorry if I missed this on the website or the documentation but will the associated code be open-sourced at some point? Of course the recommendations by officials and the ERNW penetration tests sound great, but there is nothing like the actual code to gain trust ;-).

**Comment by lesmoureal** [comment]
will happen I think. don‘t know when. i“m not the developer

-Smudo
(mit Handy verfasst)

**Reply by petrarca-arezzo** [reply]
Fully agree. An app like this must be open source so one could check if the promises made on data safety are real and honest or not. As long as it's not open source one should not trust it.

**Comment by philipp-berger** [comment, marked as answer]
Since the roadmap to publishing the code base of Luca crystallized. We can now confirm that the whole code base will be accessible until End of March.
This will be a stepwise process beginning with this security concept project and continuing with the native iOS, Android, WebApps and the backend code.

**Reply by reneme** [reply]
We're currently in the process of publishing our source code. See here: https://gitlab.com/lucaapp

**Comment by ostarke** [comment]
Hi all,  i appreciate gaining access to the source. For me the most important part of the app is the interface to the health care systems. It is no national advantage to have a single app ( and no rocket sience ;-) but to have a single common interface (API) for all apps to communicate with the healt care systems. If this would be available it gave a big boost to innovation. Since we are developing mobile apps since 1995 we know that the reason for highly innovative ideas not being brought to market has mostly been the missing possibility to communicate with  other systems. 
We are using Qt framework for app development. Perhaps it would make sense for us to port the code to Qt. The platform abstraction framework reduces the project cost because you only need to maintain one source code regardless of the operating system the app will be build for (iOs, Andoird,Windows,Linux,...) So, having access to the the code, innovation can continue ... 
BR
Oliver Starke
